tests/test_utils.py

```diff
@@ -1,4 +1,38 @@
-from mapbox_tilesets.utils import validate_tileset_id
+import os
+from mapbox_tilesets.utils import _get_session, _get_token, validate_tileset_id
+
+
+def test_get_session():
+    s = _get_session("my_application", "1.0.0")
+    assert "user-agent" in s.headers
+    assert s.headers["user-agent"] == "my_application/1.0.0"
+
+
+def test_get_token_parameter():
+    token = "token-parameter"
+    assert token == _get_token(token)
+
+
+def test_get_token_environment_variables():
+    token1 = "token-environ1"
+    token2 = "token-environ2"
+    os.environ["MAPBOX_ACCESS_TOKEN"] = token1
+    assert token1 == _get_token(None)
+    del os.environ["MAPBOX_ACCESS_TOKEN"]
+    os.environ["MapboxAccessToken"] = token2
+    assert token2 == _get_token(None)
+    del os.environ["MapboxAccessToken"]
+
+
+def test_get_token_errors_without_token():
```

Comment:
As an alternative, you can use `pytest.raises` to neatly assert on the type of error thrown as well as the message https://docs.pytest.org/en/stable/assert.html#assertions-about-expected-exceptions